jscolor.js

```diff
@@ -527,17 +527,18 @@ var jsc = {
 	onDocumentMouseDown : function (e) {
 		if (!e) { e = window.event; }
 		var target = e.target || e.srcElement;
-
-		if (target._jscLinkedInstance) {
-			if (target._jscLinkedInstance.showOnClick) {
-				target._jscLinkedInstance.show();
-			}
-		} else if (target._jscControlName) {
-			jsc.onControlPointerStart(e, target, target._jscControlName, 'mouse');
-		} else {
-			// Mouse is outside the picker controls -> hide the color picker!
-			if (jsc.picker && jsc.picker.owner) {
-				jsc.picker.owner.hide();
+		if (!target.disabled) {
+			if (target._jscLinkedInstance) {
+				if (target._jscLinkedInstance.showOnClick) {
+					target._jscLinkedInstance.show();
+				}
+			} else if (target._jscControlName) {
+				jsc.onControlPointerStart(e, target, target._jscControlName, 'mouse');
+			} else {
+				// Mouse is outside the picker controls -> hide the color picker!
+				if (jsc.picker && jsc.picker.owner) {
+					jsc.picker.owner.hide();
+				}

```

Comment:
Instead of wrapping all of this code in the `if (!disabled)` check, why not just do this at the top, and leave the rest of the code untouched?

    if (target.disabled) {
        return;
    }